spec/index.html

```diff
@@ -500,9 +500,13 @@ <h3>Blank nodes</h3>
       to the universe IR of I.
       Define the mapping [I+A] as:</p>
       <ul>
-	<li> [I+A](x)=I(x) when x is a <a>name</a>, </li>
-	<li> [I+A](x)=A(x) when x is a blank node, </li>
-	<li> [I+A](x)= RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) )  where x.s, x.p, and x.o are the first, second, and third components of x, respectively. </li>
+	<li> [I+A](x)=I(x) when x is a <a>name</a>; </li>
+	<li> [I+A](x)=A(x) when x is a blank node; </li>
+	<li> [I+A](x)= RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) ) when x is a triple term, where x.s, x.p, and x.o are the first, second, and third components of x, respectively; </li>
+	<li> [I+A](x)=true when x is a triple and [I+A](p) is in IP and the pair <[I+A](s),[I+A](o)> is in IEXT([I+A](p));
+	<li> [I+A](x)=false when x is a triple, otherwise;
+	<li> [I+A](x)=false if x is an RDF graph and [I+A](x')=false for some triple x' in x;
+        <li> [I+A](x)=true if x an RDF graph, otherwise
```

Comment:
Something has gone wrong with the HTML here.

Is this what was intended?

```suggestion
	<li> [I+A](x)=false if x is an RDF graph and [I+A](x')=false for some triple x' in x;</li>
    <li> [I+A](x)=true if x an RDF graph, otherwise</li>
```

Reply:
* _Why do some conditional phrases use `when` and others user `if`?_

  It seems to me that consistent phrasing should be used throughout this list, and throughout similar lists elsewhere in this document.

* _Why do some (multiple!) list items end with `otherwise`, sometimes followed by `;` and sometimes not?_

  It seems to me that each item in this unordered list should be disjoint from all other items in this list, and that the join of all items should encompass all possible conditions.